tests/integration/add_snippets_spec.lua

```diff
@@ -291,6 +303,22 @@ describe("add_snippets", function()
 			{2:-- INSERT --}                                      |]],
 		})
 
+		-- Test to make sure that autosnippets do not get triggered while pasting in insert mode.
+		feed("<ESC>dd") -- clear line
+		helpers.feed_command("set paste") -- disable autosnippets
```

Comment:
Oh, why does `set paste` disable autosnippets? Had a cursory read of the help-section on it, but nothing in there caught my eye